index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Style.css">
+    <title>Document</title>
```

Comment:
If you add a suitable title would be perfect.

Reply:
- [ ] It would be a good example to give a kind code review for students and more explanation of how the changes will benefit the project.

Reply:
please use a suitable title in your project. A suitable title makes your reader clear about your project.